src/pyhf/infer/calculators.py

```diff
@@ -186,7 +195,15 @@ def __init__(
         self.par_bounds = par_bounds or pdf.config.suggested_bounds()
         self.fixed_params = fixed_params or pdf.config.suggested_fixed()
 
-        self.qtilde = qtilde
+        # handle deprecation for qtilde gracefully
```

Comment:
if this is for 0.6.0 shouldn't just make a hard cut? most people I think don't use this ctor directly but rather through our other APIs

Reply:
this is a good point. i guess i wasn't so sure at first, but it's a good point.